rake_helper.rb

```diff
@@ -0,0 +1,130 @@
+require 'aws-sdk-lambda'
+require 'aws-sdk-cloudwatchevents'
+require 'yaml'
+
+class RakeHelper
+
+  attr_reader(
+    :travis_branch,
+    :aws_access_key_id,
+    :aws_secret_access_key,
+    :aws_configuration,
+    :lambda_client,
+    :yaml,
+    :lambda_config,
+    :function_name,
+    :event
+  )
+
+  def initialize
+    @travis_branch = ENV["TRAVIS_BRANCH"].upcase
```

Comment:
I would add a conditional to check here to check if the branch in `main` or `master` to make this a little more generic

`@travis_branch = ['MAIN', 'MASTER'].include? @travis_branch ? 'PRODUCTION' : @travis_branch`

Reply:
good point, added